examples/tool.xml

```diff
@@ -11,20 +15,19 @@
   </stdio>
   <version_command>aragorn.exe --version</version_command>
   <command><![CDATA[aragorn.exe $flag
--float $float
+float_section $float_section
 cond $cond
 -i$int_min,$int_max
 
 $posint
--output $output]]></command>
+select_local $select_local]]></command>
   <inputs>
     <param argument="-flag" checked="false" help="Flag help" label="Flag label" name="flag" type="boolean" truevalue="-flag" falsevalue=""/>
-    <param argument="-float" help="Float help" label="Float label" name="float" type="float" value="0"/>
+    <section name="float_section" title="Float section">
+      <param argument="-float" help="Float help" label="Float label" name="float" type="float" value="0"/>
+    </section>
     <conditional argument="cond" label="Conditional" name="cond">
-      <param argument="Select" label="Author did not provide help for this parameter... " name="Select" type="select">
-        <option value="bye">2</option>
-        <option value="hi">1</option>
-      </param>
+      <param argument="Select" label="Author did not provide help for this parameter... " name="Select" type="select"/>
```

Comment:
Looks like something went wrong with the select since it isn't outputting the options any more